commit 8ac37e786d36252404c0cd64e7939aed516d9820
Author: Mike Hiretsky <mh@calculate.ru>
Date:   Mon Mar 28 16:17:24 2011 +0400

    Fix fault by get cl_builder_path in scratch mode.

diff --git a/pym/cl_fill_builder.py b/pym/cl_fill_builder.py
index 1435bf6..2b4d512 100644
--- a/pym/cl_fill_builder.py
+++ b/pym/cl_fill_builder.py
@@ -162,7 +162,7 @@ class fillVars(object, varsShare):
                     inidata = iniParser(inifile)
                     res = inidata.getVar("assemble","cl_assemble_path",
                                          checkExistVar=True)
-                    if res[0]:
+                    if res and res[0]:
                         return _toUNICODE(res[1]).encode('utf-8')
             return "/mnt/builder"
         return "/"
